setup.py

```diff
@@ -13,8 +13,10 @@
 import platform
 import shutil
```

Comment:
Note that both let and const are block-scoped, whereas var is function-scoped. test

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Unreadable, Secure, Efficient